pkg/apis/v1alpha1/labels.go

```diff
@@ -48,6 +48,7 @@ func init() {
 }
 
 var (
+	CapacityTypeKey                 = "karpenter.sh/capacity-type"
```

Comment:
there is one definition in `vendor/sigs.k8s.io/karpenter/pkg/apis/v1beta1/labels.go:42`